Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polyline.h

```diff
@@ -1210,16 +1210,87 @@ class Triangulate_hole_polyline {
  ************************************************************************/
 // /!\ points.first == points.last
 
+template <typename Traits>
+bool is_planar_2(
```

Comment:
Would this function not be of general interest? @sloriot

Reply:
Yes but I don't know where we should put it.